package/MDAnalysis/coordinates/XTC.py

```diff
@@ -26,15 +37,15 @@ class XTCWriter(XDRBaseWriter):
     they require significantly less disk space and the loss of precision is
     usually not a problem.
 
-    Parameter
-    ---------
-    filename : str
+    Parameters
+    ----------
+    filename: str
```

Comment:
the space is also there in napoleons own docs.

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

maybe you can update your version or write a bug report to them.